clang/cmake/caches/Fuchsia.cmake

```diff
@@ -70,10 +70,6 @@ foreach(variable ${_FUCHSIA_BOOTSTRAP_PASSTHROUGH})
   endif()
 endforeach()
 
-if(WIN32)
-  set(LLVM_USE_CRT_RELEASE "MT" CACHE STRING "")
```

Comment:
Same here - we should probably set `CMAKE_MSVC_RUNTIME_LIBRARY` instead of just removing the setting?

Reply:
Same as above, maybe Line57 is enough?